apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -6811,6 +6811,9 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
             }
 
             data.state.code = (isCompanyAddress) ? address.stateCode : (address.state) ? address.state.code : "";
+            data.stateCode = data.state.code;
```

Comment:
Add optional chaining operator on state (`?`) in `data.state.code` and `data.state.name`